### DISCUSSION TITLE
False positive CVE-2022-4065 reported against org.testng:testng (testng-7.7.1.jar)

### DISCUSSION THREAD

**Discussion by navzen2000** [aquasecurity/trivy#4801] [False Detection]
### IDs

CVE-2022-4065

### Description

Trivy reported a false positive:
CVE-2022-4065
on
org.testng:testng (testng-7.7.1.jar)   

### Reproduction Steps

```bash
1.run trivy on container with org.testng:testng (testng-7.7.1.jar)  
2.
3.
...
```

### Target

Container Image

### Scanner

Vulnerability

### Target OS

Linux

### Debug Output

```bash
{
          "VulnerabilityID": "CVE-2022-4065",
          "PkgName": "org.testng:testng",
          "PkgPath": "apps/libs/testng-7.7.1.jar",
          "InstalledVersion": "7.7.1",
}
```

### Version

```bash
Version: 0.42.1
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-07-07 06:11:20.146872756 +0000 UTC
  NextUpdate: 2023-07-07 12:11:20.146872556 +0000 UTC
  DownloadedAt: 2023-07-07 06:59:29.390528351 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-07-07 00:55:37.190607976 +0000 UTC
  NextUpdate: 2023-07-10 00:55:37.190607376 +0000 UTC
  DownloadedAt: 2023-07-07 03:54:31.1043294 +0000 UTC
```

### Checklist

- [X] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/dev/community/contribute/discussion/#false-detection)
- [X] Ran Trivy with `-f json` that shows data sources and confirmed that the security advisory in data sources was correct

**Comment by DmitriyLewen** [comment]
Hello @navzen2000 
Thanks for your report!

We know about this problem.
Issue for GitLab already created - https://github.com/aquasecurity/trivy/discussions/4799#discussioncomment-6399751

Best Regards, Dmitriy
